build.xml

```diff
@@ -442,6 +442,8 @@
             <fileset file=".ant-targets-build.xml"/>
             <fileset file=".run.sh"/>
             <fileset file="release.zip"/>
+            <fileset file="spotbugs.html"/>
+            <fileset file="spotbugs-check.html"/>
```

Comment:
This should be removed in `clean`, not `realclean`, or the removal of other spotbugs output should be removed from `clean`.

Reply:
Thanks, will fix.